packages/ui/src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -174,15 +174,13 @@ class Container extends React.Component<Props, ComponentState> {
                         ? NONE_RESOLVER_KEY
                         : nextProps.entity.resolverType
 
-                    const isMultiValueDisabled = this.isMultiValueDisabled(nextProps.entity, this.props.actions)
-
                     this.setState({
                         entityNameVal: nextProps.entity.entityName,
                         entityTypeVal: entityType,
                         entityResolverVal: resolverType,
                         isPrebuilt: isPrebuilt,
                         isMultivalueVal: nextProps.entity.isMultivalue,
-                        isMultiValueDisabled,
+                        isMultiValueDisabled: false,
```

Comment:
Based on comment above it seems it's not confirms how to deal with this, but if this property is always `false` then there is no need for the property. Can remove.

Reply:
I put it back.